managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -1746,7 +1746,7 @@ public Result healthCheck(UUID customerUUID, UUID universeUUID) {
     try {
       List<HealthCheck> checks = HealthCheck.getAll(universeUUID);
       for (HealthCheck check : checks) {
-        detailsList.add(Json.stringify(Json.parse(check.detailsJson)));
+        detailsList.add(check.detailsJson.toString());
```

Comment:
Is there a unit test that reads this response so that we know its not broken.

Reply:
No

Reply:
Either add unitest tests or run the platform create universe and check if health checking functionality works